### PR TITLE
Fixed test style and updated image

### PR DESCRIPTION
**Pull request by johngschulz**
This updated the test style to use Bitstream Sera Sans font to address visual test issues resulting from unregistered fonts.
commented out system interactive, true property

### CODE REVIEW

File: modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/transform/VisualTransformerTest.java [hunk 1]
```diff
@@ -118,7 +118,7 @@ public void setUp() throws Exception {
         		 .openStream()));
 
         // UNCOMMENT THE BELOW LINE TO DISPLAY VISUAL TESTS
-         System.setProperty("org.geotools.test.interactive", "true");
+        // System.setProperty("org.geotools.test.interactive", "true");
```
Comment (jodygarnett) [inline modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/transform/VisualTransformerTest.java]: I am not sure these are required, we have instructions in our docs about using -D command line options for interactive tests. You should be able to do the same thing from your IDE.

**Comment by jodygarnett** [comment]
Small question good to merge; thanks for the debugging

File: modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/transform/VisualTransformerTest.java (hunk 1 above)
Comment (johngschulz) [reply modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/transform/VisualTransformerTest.java]: These were in place in other visual test as well, I think it's just there to make testing easier.  At any rate I had it uncommented while working through it and just remembered to re-comment out.

Comment (aaime) [reply modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/transform/VisualTransformerTest.java]: I use that a lot but don't put it in the code anymore, all launch configurations for rendering unit tests in my IDE have ``-Dorg.geotools.image.test.interactive=true``, this way one cannot forget uncommenting.

**Pull request merged**